.github/workflows/license.yml

```
@@ -0,0 +1,35 @@
+name: Update License
+
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+
+
+    steps:
+      # Checks-out repo
+      - uses: actions/checkout@v2
+
+      #This uses sed to change the year
+      - name: Edit the license file
+        run: |
+          cd ./actions/license
+          python update.py
+
+      - name: Commit the changes
+        #This makes sure the workflow doesn't fail if there's nothing to be committed
+        continue-on-error: true
+        run: |
+          git config --global user.email "<>"
+          git config --global user.name "Github Actions Bot"
+          git add .
+          git commit -m 'Update license year'
+          git push
```

```suggestion
      - name: Commit the changes
        run: |
          git config --global user.email "<>"
          git config --global user.name "Github Actions Bot"
          git add .
          git commit -m 'Update license year' || true # pass is there is nothing to commit
          git push
```

this would hide errors on push which might not be expected behavior. I would rather apply it only to the commit line